From patchwork Fri Apr 17 00:12:30 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Fabio Estevam <festevam@gmail.com>
X-Patchwork-Id: 1271951
Return-Path: <u-boot-bounces@lists.denx.de>
X-Original-To: incoming@patchwork.ozlabs.org
Delivered-To: patchwork-incoming@bilbo.ozlabs.org
Authentication-Results: ozlabs.org;
 spf=pass (sender SPF authorized) smtp.mailfrom=lists.denx.de
 (client-ip=85.214.62.61; helo=phobos.denx.de;
 envelope-from=u-boot-bounces@lists.denx.de; receiver=<UNKNOWN>)
Authentication-Results: ozlabs.org;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: ozlabs.org; dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.a=rsa-sha256
 header.s=20161025 header.b=mVCLNe6i; dkim-atps=neutral
Received: from phobos.denx.de (phobos.denx.de [85.214.62.61])
 (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
 key-exchange X25519 server-signature RSA-PSS (4096 bits))
 (No client certificate requested)
 by ozlabs.org (Postfix) with ESMTPS id 493Gl272g2z9sRN
 for <incoming@patchwork.ozlabs.org>; Fri, 17 Apr 2020 10:13:02 +1000 (AEST)
Received: from h2850616.stratoserver.net (localhost [IPv6:::1])
 by phobos.denx.de (Postfix) with ESMTP id E10D5819FD;
 Fri, 17 Apr 2020 02:12:46 +0200 (CEST)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=u-boot-bounces@lists.denx.de
Authentication-Results: phobos.denx.de; dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.b="mVCLNe6i";
 dkim-atps=neutral
Received: by phobos.denx.de (Postfix, from userid 109)
 id F2250819F3; Fri, 17 Apr 2020 02:12:21 +0200 (CEST)
X-Spam-Checker-Version: SpamAssassin 3.4.2 (2018-09-13) on phobos.denx.de
X-Spam-Level: 
X-Spam-Status: No, score=-2.0 required=5.0 tests=BAYES_00,DKIM_SIGNED,
 DKIM_VALID,DKIM_VALID_AU,FREEMAIL_FROM,SPF_HELO_NONE autolearn=ham
 autolearn_force=no version=3.4.2
Received: from mail-qv1-xf41.google.com (mail-qv1-xf41.google.com
 [IPv6:2607:f8b0:4864:20::f41])
 (using TLSv1.3 with cipher TLS_AES_128_GCM_SHA256 (128/128 bits))
 (No client certificate requested)
 by phobos.denx.de (Postfix) with ESMTPS id 640F8805A0
 for <u-boot@lists.denx.de>; Fri, 17 Apr 2020 02:12:12 +0200 (CEST)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=festevam@gmail.com
Received: by mail-qv1-xf41.google.com with SMTP id s18so94793qvn.1
 for <u-boot@lists.denx.de>; Thu, 16 Apr 2020 17:12:12 -0700 (PDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025; 
 h=from:to:cc:subject:date:message-id:in-reply-to:references;
 bh=6W7EQQ1hvBkWfx+sM9RXzYVn48sPcAdI/6HYww6Sxpw=;
 b=mVCLNe6icHjcZ2UgEjY+tME5sFqR3o8yg2JErpGx5U8fV/s6T5OVDwZzYaB4LH+e6U
 UNqtSpB9NqoNG02ny2Q5cFiqKo8MLAUw+nFVqAFK6WOdrIhgSFWhNWZfbCAAYMXsXsFZ
 Eqlbn4tkPnldajfG7HhnDaznC+fI3wjd/AVzhOeHe3mXYRKSzMmnpnHnNS4EJ6fO3bun
 dNudwShmyWKWdYV/1h06yoPxPazLMV+nGueNp57jxKaYlrIFIQ517ujx6gaEJ0tOhCgV
 eKRnTnXsySXJw59Y7HyHjo0t7Qm2+sdEN7MjvApHWf/dJLYTw7w9vUS4GSeM9qnrMpOV
 nnrg==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20161025;
 h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
 :references;
 bh=6W7EQQ1hvBkWfx+sM9RXzYVn48sPcAdI/6HYww6Sxpw=;
 b=oqtxasHhtJyGW2lO18tvcWaG2wwWZd2X/HZrbDMPRTVMDo8jBgX3Ea8aSXLs8ojVr/
 ZVb8twBfE7Unlz/srOmMn6TK+PLnCyc6cukaQe9mELOgmTcb9qh6HSaL25NzCwd6OKaG
 EcipB+3o64dqcJALw7MkAtFz4j7yIUuX/w6QeH8o66gSXgRoyV5X8TMlIGDXVXuBIY4X
 KLULaXv/LWl3U+AA25rudp0TUj3mwPmf59/B7TvrMD2/TZgPSL88kcUuPazrUwGu4dr9
 MzVhN88HwOvcSrA2mQ3FMIWHeW/zwD+yrIUjZxjrDFTZ7ORvN8Zpjd3aw0se6Eg4zjS1
 SQIA==
X-Gm-Message-State: AGi0Puaov0D9+t15xSQNb0dQEnJQFH4ATUAmnkLmcUxKhk8i1qPtjtL4
 dsN+UXaOKGbJkUlPW2BD8wc=
X-Google-Smtp-Source: APiQypKSQxpfKwaAW+xOyKDp81dsX+z2Buq5L3ce0i1oekP0V2n46R+5FNuKeAJVEcZkkdEdmkeGoQ==
X-Received: by 2002:ad4:46b4:: with SMTP id br20mr282264qvb.62.1587082331257; 
 Thu, 16 Apr 2020 17:12:11 -0700 (PDT)
Received: from localhost.localdomain ([2804:14c:482:271:809b:81f0:1d41:9d6b])
 by smtp.gmail.com with ESMTPSA id
 x5sm16987996qtm.81.2020.04.16.17.12.08
 (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
 Thu, 16 Apr 2020 17:12:10 -0700 (PDT)
From: Fabio Estevam <festevam@gmail.com>
To: sbabic@denx.de
Cc: hs@denx.de, derek@ihtfp.com, u-boot@lists.denx.de,
 Fabio Estevam <festevam@gmail.com>
Subject: [PATCH 4/5] wandboard: Remove CONFIG_DISPLAY_BOARDINFO_LATE
Date: Thu, 16 Apr 2020 21:12:30 -0300
Message-Id: <20200417001231.11406-4-festevam@gmail.com>
X-Mailer: git-send-email 2.17.1
In-Reply-To: <20200417001231.11406-1-festevam@gmail.com>
References: <20200417001231.11406-1-festevam@gmail.com>
X-BeenThere: u-boot@lists.denx.de
X-Mailman-Version: 2.1.30rc1
Precedence: list
List-Id: U-Boot discussion <u-boot.lists.denx.de>
List-Unsubscribe: <https://lists.denx.de/options/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=unsubscribe>
List-Archive: <https://lists.denx.de/pipermail/u-boot/>
List-Post: <mailto:u-boot@lists.denx.de>
List-Help: <mailto:u-boot-request@lists.denx.de?subject=help>
List-Subscribe: <https://lists.denx.de/listinfo/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=subscribe>
Errors-To: u-boot-bounces@lists.denx.de
Sender: "U-Boot" <u-boot-bounces@lists.denx.de>
X-Virus-Scanned: clamav-milter 0.102.2 at phobos.denx.de
X-Virus-Status: Clean

Since we are using revD1 device tree for all board revisions, the
following can be seen on a revB1 board:

Model: Wandboard i.MX6 Quad Board revD1
Board: Wandboard rev B1

To avoid such confusing messages, disable CONFIG_DISPLAY_BOARDINFO_LATE.

Signed-off-by: Fabio Estevam <festevam@gmail.com>
---
 configs/wandboard_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/wandboard_defconfig b/configs/wandboard_defconfig
index ee70758f3a..f9a5fe479b 100644
--- a/configs/wandboard_defconfig
+++ b/configs/wandboard_defconfig
@@ -29,7 +29,6 @@ CONFIG_BOOTCOMMAND="run findfdt; run finduuid; run distro_bootcmd"
 CONFIG_SYS_CONSOLE_IS_IN_ENV=y
 CONFIG_SYS_CONSOLE_OVERWRITE_ROUTINE=y
 # CONFIG_DISPLAY_BOARDINFO is not set
-CONFIG_DISPLAY_BOARDINFO_LATE=y
 CONFIG_BOUNCE_BUFFER=y
 CONFIG_BOARD_EARLY_INIT_F=y
 CONFIG_SPL_SEPARATE_BSS=y
